### PR TITLE
MULTIARCH-4696: fix copy kubeconfig to root kube folder

### PR DESCRIPTION
**Pull request by pkenchap**
copy kubeconfig to /root/.kube/config .

### CODE REVIEW

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 1]
```diff
@@ -379,6 +379,9 @@ function build_upi_cluster() {
 
     echo "Copying the terraform.tfstate"
     cp "${IBMCLOUD_HOME}"/ocp4-upi-powervs/terraform.tfstate "${SHARED_DIR}"/terraform.tfstate
+    if [ ! -f "/root/.kube/config" ]; then
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:382]: failure was in an earlier step... still debugging in this PR / rehearsal
root copy failure isn't on this system - it's on the bastion.

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 2]
```diff
@@ -379,6 +379,9 @@ function build_upi_cluster() {
...
+        cp "${IBMCLOUD_HOME}"/ocp-install-dir/kubeconfig "/root/.kube/config"
+    fi
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:384]:
```suggestion
    if [ ! -f "${SHARED_DIR}"/kubeconfig ]
    then
        echo "kubeconfig not found install failed"
        exit 7
    fi
```

Comment (pkenchap) [reply ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh]: Sure , addressed , thanks .

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.18-ocp-e2e-powervs-ovn-multi-p-px

**Review by prb112 — APPROVED** [review]
/lgtm

**Pull request merged**